js/src/product-feed/issues-table-card/index.js

```diff
@@ -80,20 +42,24 @@ const actions = (
 	</HelpPopover>
 );
 
+/**
+ * The card rendering the Merchant Center Issues in the Product feed. It doesn't render if there is no issues.
+ * It uses useMCIssues for getting the total of issues.
+ *
+ * @see useMCIssues
+ * @see IssuesTypeNavigation
+ * @see IssuesTable
+ * @return {JSX.Element|null} A Card with a Header, a Navigation and a table with issues
+ */
 const IssuesTableCard = () => {
-	const [ page, setPage ] = useState( 1 );
-	const { hasFinishedResolution, data } = useAppSelectDispatch(
-		'getMCIssues',
-		{
-			page,
-			per_page: ISSUE_TABLE_PER_PAGE,
-		}
-	);
+	const issues = useMCIssues();
 
-	const handlePageChange = ( newPage, direction ) => {
-		setPage( newPage );
-		recordTablePageEvent( 'issues-to-resolve', newPage, direction );
-	};
+	// We don't want to render if no issues are found
+	if (
+		! issues[ ISSUE_TYPE_PRODUCT ]?.data?.total &&
+		! issues[ ISSUE_TYPE_ACCOUNT ]?.data?.total
+	)
+		return null;
```

Comment:
Thats a very good question. Let me escalate it to design to see what they think. At the beginning they wanted to show a message like "congrats, no issues left" but we didn't talk about the default tab yet.

However, since this is not going to production until we finish all the feature, and the design has not being validated 100% we can perform that change in further issues if needed.

Reply:
Update: 

After asking design team they approve this suggestion with the status "nice to have". I will create a new issue soon in order to implement this. 

pcTzPl-da-p2#comment-430

Reply:
Thanks @puntope!